security/vertx-jwt/src/test/java/io/quarkus/ts/security/vertx/AbstractCommonIT.java

```diff
@@ -43,16 +43,16 @@ public abstract class AbstractCommonIT {
     static DefaultService redis = new DefaultService().withProperty("ALLOW_EMPTY_PASSWORD", "YES");
 
     @QuarkusApplication(certificates = {
-            @Certificate(format = Certificate.Format.PEM, prefix = VALID_PEM_PREFIX),
-            @Certificate(format = Certificate.Format.PEM, prefix = INVALID_PEM_PREFIX)
+            @Certificate(format = Certificate.Format.PEM, prefix = VALID_PEM_PREFIX, configureHttpServer = true, useTlsRegistry = false),
```

Comment:
@michalvavrik why do we use `configureHttpServer`? This test doesn't use https connection

Reply:
Intention of my commit is to keep status quo. I don't look what this test does or not, I only wanted to keep previous behavior. So previously this property was implicitly true, now it is not. I would not dare to change functionality in dependabot bump. that is why.

I agree this should be drop it. I'll look into it.

Reply:
by implicitly true I mean these properties were set and the flag didn't exist.

Reply:
OK, thank you, "it was like that by default" is fine with me.

For context: without these options the test failed, but I was not sure, if we need to use `configureHttpServer` or, say, `configureTruststore`

Reply:
that is because of this stupid condition https://github.com/quarkus-qe/quarkus-test-framework/blob/main/quarkus-test-core/src/main/java/io/quarkus/test/security/certificate/CertificateBuilder.java#L49 I will tweak it in FW but let's not wait for FW PR and release, it could take a while. Workaround this for now: https://github.com/quarkus-qe/quarkus-test-suite/pull/1976

Reply:
Certificate generation is not required for OpenShift serving certificates, so I added there condition to run less code.